src/documents/InfusionEventSystem.md

```diff
@@ -88,7 +91,15 @@ In general you shouldn't fire any of a component's events unless invited to by i
 
 ## Registering a listener to an event
 
-Both as part of defaults, and also as supplied instantiation options, a fluid component can accept a structure named `listeners`. In the simplest form, the keys of the `listeners` structure are taken from the set of `events` present in the component's [Grade](ComponentGrades.md), and the values are either single listener specifications or arrays of listener specifications. A ***listener specification*** can take a number of forms - either being written as a simple String or Function, or as a full JSON object.The standard way of declaring a listener using Infusion's [IoC](to-do/IoCInversionOfControl.md) system is to supply the name of a global function using the member **`funcName`** or to supply a [reference](IoCReferences.md) to a function handle (usually an [Invoker](Invokers.md)) somewhere in the component tree using the member **`func`**. If your listener would like to receive different arguments than the ones that the event was fired with, you can supply references to these using the member **`args`**. You can consult the page [Event injection and boiling](EventInjectionAndBoiling.md) for the use of these more complex listener specifications. If you are happy with the existing arguments you can write a simple definition as a String or Function holding the value that would have been written in `func/funcName`. Here is a simple example of a listener definition, expanding our example from earlier:
+Both as part of defaults, and also as supplied instantiation options, an Infusion component can accept a structure named `listeners`. 
+In the simplest form, the keys of the `listeners` structure are taken from the set of `events` present in the component's [Grade](ComponentGrades.md), and the values are either single listener specifications or arrays of listener specifications. 
+A ***listener specification*** can take a number of forms - either being written as a simple String or Function, or as a full JSON object.
+The standard way of declaring a listener using Infusion's [IoC](to-do/IoCInversionOfControl.md) system is to supply the name of a global function using the member **`funcName`** or 
```

Comment:
These two possibilities would be better presented in a table – more easy to process for the reader.